docs/apis/core/hooks/index.md

```diff
@@ -158,9 +158,9 @@ namespace mod_activity\hook;
 
 #[\core\attribute\label('Hook dispatched at the very end of installation of mod_activity plugin.')]
 #[\core\attribute\tags('installation')]
-class installation_finished implements \core\hook\described_hook {
+final class installation_finished {
     public function __construct(
-        public function string $version,
+        public string $version,
```

Comment:
Bah - this was mean to be using readonly properties, but I mistyped `function` instead.
```suggestion
        public readonly string $version,
```